src/app/pages/home/home.page.html

```diff
@@ -13,23 +13,8 @@
   <ion-card class="welcome-card">
     <ion-img class="logo" src="assets/aerogear.png"></ion-img>
     <ion-card-header>
-      <ion-card-title text-center>AeroGear Mobile Services Showcase Application</ion-card-title>
+      <ion-card-title text-center>AeroGear Mobile Services Showcase</ion-card-title>
```

Comment:
not sure if we really need this title. Will it look better if we remove this completely?